features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/BotDetection/EmailConfig/ListEmails.jsx

```diff
@@ -0,0 +1,244 @@
+                        + 'authorization. WSO2 API Manager\'s bot detection mechanism traces and '
+                        + 'logs details of such unauthorized API calls and sends notifications '
+                        + 'in this regard via emails. Thereby this helps Publishers to protect '
+                        + 'their data from bot attackers and improve the security of their data.'
+                        + '{breakingLine}{breakingLine}'
+                        + 'You can add, delete and see the list of registered emails here.'
+                    }
+                />
+            </Typography>
+        ),
+        title: (
+            <Typography gutterBottom variant='h5' component='h2'>
+                <FormattedMessage
+                    id='AdminPages.BotDetection.Email.List.empty.title'
+                    defaultMessage='Notification receiving Emails'
+                />
+            </Typography>
+        ),
+    };
+
+    const analyticsDisabledEmptyBoxProps = {
+        content: (
+            <Typography variant='body2' color='textSecondary' component='p'>
+                <FormattedMessage
+                    id='AdminPages.BotDetection.Email.List.analytics.disabled.empty.content'
+                    values={{
+                        breakingLine: <br />,
+                    }}
+                    defaultMessage={
+                        'If you enable WSO2 API Manager Analytics with WSO2 API Manager, '
+                    + 'you can enable email notifications for all unauthorized API calls that '
+                    + 'you receive and also view the bot detection data easily via the Admin Portal.'
+                    + '{breakingLine}{breakingLine}'
+                    + 'Follow documentations on help to enable Analytics and get started.'
+                    }
+                />
+            </Typography>
+        ),
+        title: (
+            <Typography gutterBottom variant='h5' component='h2'>
+                <FormattedMessage
+                    id='AdminPages.BotDetection.Email.List.analytics.disabled.empty.title'
+                    defaultMessage='Analytics disabled!'
+                />
+            </Typography>
+        ),
+    };
+
+    if (isAnalyticsEnabled === undefined) {
+        return (
+            <ContentBase pageStyle='paperLess'>
+                <InlineProgress />
+            </ContentBase>
+        );
+    }
+
+    return (isAnalyticsEnabled ? (
+        <ListBase
+            columProps={columProps}
+            pageProps={pageProps}
+            addButtonProps={addButtonProps}
+            searchProps={searchProps}
+            emptyBoxProps={emptyBoxProps}
+            apiCall={apiCall}
+            EditComponent={AddEmails}
+            DeleteComponent={DeleteEmail}
+        />
+    )
+        : (
+            <ListBase
+                pageProps={pageProps}
+                emptyBoxProps={analyticsDisabledEmptyBoxProps}
```

Comment:
Shall we use a separate `EmptyBox` kind of component to show a message when there are no elements ?

Reply:
Fixed this and other occurrence with commit: https://github.com/wso2/carbon-apimgt/pull/8591/commits/8c5e0ad8207b54190c0c22e247b8be47943105d4